toolchain/Dockerfile

```diff
@@ -90,9 +92,8 @@ RUN export DEBIAN_FRONTEND=noninteractive \
         libssl-dev \
         libtool-bin \
     && cd /root \
-    && git clone https://git.yoctoproject.org/git/opkg \
+    && git clone --depth 1 --branch v0.4.5 https://git.yoctoproject.org/git/opkg \
```

Comment:
Updated opkg a bit as well, 0.4.4->0.4.5.

We can bump it all the way to 0.6.2, if this won't break anything.

Reply:
My reply to this was missing for some reason. The version of opkg in entware is from 2022-02-24, so we should try to match its version here.

Reply:
Entware uses [opkg-lede](https://git.openwrt.org/?p=project/opkg-lede.git;a=summary), which is a (lesser maintained?) fork of [opkg](https://git.yoctoproject.org/opkg/log/?h=v0.7.0). For compatibility we can it as well, not sure what issues that might bring now.

Reply:
It's what we use on device, so it would be best to use the same thing. It's probably a miss that we are using version.